pkg/neorpc/result/version.go

```diff
@@ -15,12 +15,13 @@ type (
 	// Version model used for reporting server version
 	// info.
 	Version struct {
-		TCPPort   uint16   `json:"tcpport"`
-		WSPort    uint16   `json:"wsport,omitempty"`
-		Nonce     uint32   `json:"nonce"`
-		UserAgent string   `json:"useragent"`
-		Protocol  Protocol `json:"protocol"`
-		RPC       RPC      `json:"rpc"`
+		TCPPort     uint16      `json:"tcpport"`
+		WSPort      uint16      `json:"wsport,omitempty"`
+		Nonce       uint32      `json:"nonce"`
+		UserAgent   string      `json:"useragent"`
+		Protocol    Protocol    `json:"protocol"`
+		Application Application `json:"application"`
```

Comment:
Add a comment following the style of other extensions, e.g.:

https://github.com/nspcc-dev/neo-go/blob/2971b8a1c7dfc8cb895f9a60e8f1f36c60020948/pkg/neorpc/result/version.go#L54-L58